lib/CustomFields/components/FieldAccordion/view-sections/TextboxSection.js

```diff
@@ -21,9 +21,18 @@ const TextboxSection = props => (
         {props.name}
       </KeyValue>
     </Col>
+    {
+      props.helpText ? (
+        <Col xs={3}>
+          <KeyValue label={<FormattedMessage id="stripes-smart-components.customFields.helperText" />}>
+            {props.helpText}
+          </KeyValue>
+        </Col>
+      ) : null
+    }
     <Col xs={3}>
-      <KeyValue label={<FormattedMessage id="stripes-smart-components.customFields.helperText" />}>
-        {props.helpText || <NoValue />}
+      <KeyValue label={<FormattedMessage id="stripes-smart-components.customFields.required" />}>
+        {props.required ? 'Yes' : 'No'}
```

Comment:
Change `'Yes'`,  `'No'` to `<FormattedMessage id="..." />`, you need to add this strings to translations

Reply:
Fixed in new commit